### DISCUSSION TITLE
喜报

### DISCUSSION THREAD

**Discussion by boomzero** [langningchen/XMOJ-Script#195] [General]
能用bing搜到XMOJ-Script!!!  
https://www.bing.com/search?q=XMOJ-Script&qs=n&form=QBRE&sp=-1&ghc=1&lq=0&pq=xmoj-scri&sc=2-9&sk=&cvid=CF2B9AB5D52D4FE58D61E9CB5506B01C&ghsh=0&ghacc=0&ghpl=

google也行, 而且更好!
https://www.google.com.hk/search?q=XMOJ-Script&sca_esv=560895637&source=hp&ei=yXLtZMCFDKXWkPIP0_mMkAQ&iflsig=AD69kcEAAAAAZO2A2ddFKncxbGVTMAdHRse7cemTKE8O&ved=0ahUKEwiAjvrlgoGBAxUlK0QIHdM8A0IQ4dUDCAg&uact=5&oq=XMOJ-Script&gs_lp=Egdnd3Mtd2l6IgtYTU9KLVNjcmlwdDIFEAAYogQyBRAAGKIEMgUQABiiBEiPI1AAWKYhcAF4AJABAJgBxQWgAf0cqgEJMy01LjAuMi4xuAEDyAEA-AEBwgINEC4YDRiABBjHARjRA8ICBxAAGA0YgATCAgcQLhgNGIAEwgIFEAAYgATCAgsQLhivARjHARiABA&sclient=gws-wiz

**Comment by boomzero** [comment]
![clone](https://github.com/langningchen/XMOJ-Script/assets/85378277/9e3c5c0c-2d57-41ae-a5f2-5f6cd33deb9e)
庆祝本项目被clone 525次!

**Comment by langningchen** [comment]
8月28号干啥呢？clone了这么多次

**Comment by langningchen** [comment]
152个不同的人都clone了？不可能这么多

**Reply by PythonSmall-Q** [reply]
我23个不同的服务器crawer

**Reply by PythonSmall-Q** [reply]
doge

**Comment by PythonSmall-Q** [comment]
8/28我的服务器在做github存档。你的项目服务器clone出bug了任务重启了好多次

**Reply by PythonSmall-Q** [reply]
@langningchen

**Reply by PythonSmall-Q** [reply]
你要什么牌子？

**Reply by langningchen** [reply]
要么和他一样吧

**Reply by PythonSmall-Q** [reply]
那就咕咕咕了（？

**Reply by PythonSmall-Q** [reply]
你可以试试我什么牌子都能加

**Reply by PythonSmall-Q** [reply]
要不“aya so cute”？

**Reply by langningchen** [reply]
就这样吧

**Reply by PythonSmall-Q** [reply]
好的我去加咕咕咕

**Reply by langningchen** [reply]
> 要不“aya so cute”？

这个应该是你自己的牌子，而不是我的

**Reply by PythonSmall-Q** [reply]
![image](https://github.com/langningchen/XMOJ-Script/assets/106425289/570abc4c-e7a8-412c-b8ca-bb30ba99b266)
加好了

**Reply by langningchen** [reply]
好的

**Comment by langningchen** [comment]
能搜到，因为搜索引擎抓取了Github pages和Github页面，但是xmoj-bbs.tech应该抓取不到

**Reply by boomzero** [reply]
我看一下, 加一下SEO

**Reply by langningchen** [reply]
不用吧

**Comment by boomzero** [comment]
@langningchen 
![serch](https://github.com/langningchen/XMOJ-Script/assets/85378277/0dd9f2db-013c-45fa-a854-23cee6124943)

**Reply by langningchen** [reply]
OK
